charts/kong/templates/service-kong-udp-proxy.yaml

```diff
@@ -0,0 +1,14 @@
+{{- if .Values.deployment.kong.enabled }}
+{{- if and .Values.udpProxy.enabled -}}
+{{- $serviceConfig := dict -}}
+{{- $serviceConfig := merge $serviceConfig .Values.udpProxy -}}
+{{- $_ := set $serviceConfig "fullName" (include "kong.fullname" .) -}}
+{{- $_ := set $serviceConfig "namespace" (include "kong.namespace" .) -}}
+{{- $_ := set $serviceConfig "metaLabels" (include "kong.metaLabels" .) -}}
+{{- $_ := set $serviceConfig "selectorLabels" (include "kong.selectorLabels" .) -}}
+{{- $_ := set $serviceConfig "serviceName" "udp-proxy" -}}
```

Comment:
Should not serviceName include "kong-ingress" ? should they have overlap ?

Reply:
We prefix the generic name (`udp-proxy` here) with the release name in the helper template that we call on everything in `$serviceConfig` after: https://github.com/Kong/charts/blob/kong-2.1.0/charts/kong/templates/_helpers.tpl#L103